### PR TITLE
Remove needless .aws shared secret configuration

### PR DESCRIPTION
**Pull request by Fenn-CS**
So AWS Node V2 SDK is able to pick up the credentials automatically when
set as environment variables. This means setting the shared credentials in
`~/.aws` is uneccesary and even requires another environment variable
`AWS_SDK_LOAD_CONFIG` for the SDK to use it.

This also means that, currently the image/container does not make use of the
shared credentials copy from the host.

Hence,

- We don't need the .aws shared credentials template directory (now deleted)
- We dom't need `gettext` libray which is used to dynamically update the
credentials (now excluded from image)

Solves #52

### CODE REVIEW

File: .github/workflows/dockerhub.yml [hunk 1]
```diff
@@ -13,23 +13,19 @@ jobs:
...
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
       - name: Extract metadata (tags, labels) for Docker
         id: meta
-        uses: docker/metadata-action@98669ae865ea3cffbcbaa878cf57c20bbf1c6c38
+        uses: docker/metadata-action@v4
         with:
-          images: my-docker-hub-namespace/my-docker-hub-repository
+          images: permanentlegacy/my-docker-hub-repository
```
Comment (Fenn-CS) [inline .github/workflows/dockerhub.yml:24]: I am waiting for @cecilia-donnelly to give me access so I can get the right tag for this line

Comment (cecilia-donnelly) [reply .github/workflows/dockerhub.yml:24]: Based on our slack conversation and the back-end pr I thought we weren't pushing this image to dockerhub.

Comment (Fenn-CS) [reply .github/workflows/dockerhub.yml:24]: @cecilia-donnelly That's was one of the options I listed. The other one was to publish publicly without the secrets.

I don't mind removing the publish action! So, I would just do that.

**Comment by Fenn-CS** [comment]
@cecilia-donnelly 

This PR 
- Remove dockerhub publication
- Removes AWS shared credential configuration as the SDK does not need it, it picks the configuration from env vars.

**Review by liam-lloyd — APPROVED** [review]

File: .github/workflows/dockerhub.yml (hunk 1 above)
Comment (cecilia-donnelly) [reply .github/workflows/dockerhub.yml]: Thank you!

**Pull request merged**